IntelliTect.TestTools.Console/ConsoleAssert.cs

```diff
@@ -38,6 +41,7 @@ public static string Expect(string expected, Action action, bool normalizeLineEn
         /// <param name="expected">Expected "view" to be seen on the console,
         /// including both input and output</param>
         /// <param name="action">Method to be run</param>
+		[Obsolete]
```

Comment:
Odd indentation. Also Obsolete attribute should include a string indicating how people should alter their code.